strongbox-commons/src/main/java/org/carlspring/strongbox/util/LocalDateTimeInstance.java

```diff
@@ -0,0 +1,20 @@
+package org.carlspring.strongbox.util;
+
+import java.time.Clock;
+import java.time.Duration;
+import java.time.LocalDateTime;
+
+/**
+ * @author carlspring
+ */
+public class LocalDateTimeInstance
+{
+
+    public static LocalDateTime now()
+    {
+        Clock microsecondClock = Clock.tick(Clock.systemUTC(), Duration.ofNanos(1000000));
```

Comment:
they don't have `LocalDateTime.now()` in JDK 11? :)

Reply:
They do, but `now()` only returns up to milliseconds and, apparently, in JDK, the expectation is microseconds.